html final/index.html

```diff
@@ -0,0 +1,201 @@
+<!Doctype html>
+<html ln="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="styles.css">
+        <title> LIBRARY HOME - AUA's AGBU PAPAZIAN LIBRARY </title>
+        <link rel="icon" type="image/png" href="img/aua.png"/>
+    </head>
+    <body>
+        
+        <header>
+            <div class = "top-green">
+                <a href="" class = "home">AUA Home</a>
```

Comment:
Please delete the empty href="", it will reload the page when the user clicks on it
```suggestion
                <a class="home">AUA Home</a>
```